content/snippets/capacity-planning.md

```diff
@@ -0,0 +1,117 @@
+
+### Node Size
+
+The `node_size` parameter determines the size of database caches. Larger database caches decrease disk I/O requirements at a cost of higher memory requirements. Ripple recommends you always use the largest database cache your available memory can support. See the following table for recommended settings.
+
+#### Recommendation
+
+| Available RAM for `rippled` | `node_size` value | Notes                              |
+|:----------------------------|:------------------|:-----------------------------------|
+| < 8GB                       | `tiny`            | Not recommended                    |
+| 8GB                         | `low`             |                                    |
+| 16GB                        | `medium`          |                                    |
+| 32GB                        | `huge`            | Recommended for production servers |
+
+### Node DB Type
+
+The `type` field in the `node_db` section of the `rippled.cfg` file sets the type of key-value store that `rippled` uses to persist the XRP Ledger in the ledger store. You can set the value to either `rocksdb` or `nudb`.
+
+`rippled` offers a history sharding feature that allows you to store a randomized range of ledgers in a separate shard store. You may want to configure the shard store to use a different type of key-value store than the ledger store. For more information about how to use this feature, see [History Sharding](https://ripple.com/build/history-sharding/#shard-store-configuration).
+
+
+#### RocksDB vs NuDB
+RocksDB requires approximately one-third less disk storage than NuDB and provides a corresponding improvement in I/O latency. However, this comes at a cost of increased memory utilization as storage size grows. NuDB, on the other hand, has nearly constant performance and memory footprint regardless of [storage](#storage).
+
+`rippled` servers that operate as validators should keep only a few days' worth of data or less. Ripple recommends using RocksDB for validators. For all other uses, Ripple recommends using NuDB for the ledger store.
+
+RocksDB has performance-related configuration options you can modify to achieve maximum transaction processing throughput. (NuDB does not have performance-related configuration options.) Here is an example of the recommended configuration for a rippled server using RockDB:
+
+```
+[node_db]
+type=rocksdb
+open_files=512
+file_size_mb=64
+file_size_mult=2
+filter_bits=12
+cache_mb=512
+path={path_to_ledger_store}
+```
+
+### Historical Data
+
+The amount of historical data that a `rippled` server keeps online is a major contributor to required storage space. Currently, a `rippled` server stores about 12GB of data per day (2018-03-01). You can expect this amount to grow as transaction volume increases across the XRP Ledger network. You can control how much data you keep with the `online_delete` and `advisory_delete` fields.
```

Comment:
> Currently, a `rippled` server stores about 12GB of data per day (2018-03-01).

Move the date to after currently to make it clear that the date refers to what is "current" rather than the data stored on that day. That is:

> At the time of writing (2018-03-01), a `rippled` server stores about 12GB of data per day.